src/pybind/mgr/cephadm/module.py

```diff
@@ -472,6 +472,13 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             default=False,
             desc='Enable TLS security for all the monitoring stack daemons'
         ),
+        Option(
+            'upgrade_osd_flags',
+            type='str',
+            default='noout,nodeep-scrub,noscrub',
```

Comment:
my 2 cents: Is it actually a good idea to set these? I mean as long as the upgrade is reasonable fast, everything will be fine. I'm just worried about the degraded reliability of the whole cluster if the upgrade takes longer than usual. Why do we set noout for all osds, even if we don't touch them for e.g. a month if the upgrade is stuck somewhere?